code/index.html

```diff
@@ -4,34 +4,63 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <link rel="stylesheet" href="style.css" />
-    <title>Technigo Project Cluedo</title>
+    <link rel="stylesheet" href="./style.css" />
+    <title>Project Cluedo</title>
+    <link href="https://fonts.googleapis.com/css2?family=Carter+One&family=Josefin+Sans&display=swap" rel="stylesheet">
   </head>
+
   <body>
-    <section class="cards">
-      <article class="killer-deck" id="killerCard">
-        <p>KILLER</p>
-        <p id="killerName"></p>
-        <p id="killerAge"></p>
-        <p id="killerOccupation"></p>
-        <img id="killerImage" src="" height="100" />
-        <p id="killerDescription"></p>
-      </article>
-      <article class="weapon-deck" id="weaponCard">
-        <p>WEAPON</p>
-        <p id="weaponName"></p>
-        <p id="weaponWeight"></p>
-      </article>
-      <article class="room-deck" id="roomCard">
-        <p>ROOM</p>
-        <p id="roomName"></p>
-      </article>
-    </section>
-
-    <h1 id="mystery"></h1>
-
-    <button>REVEAL THE CRIME</button>
-
-    <script src="clue.js"></script>
+    <div class="main-container">
+      <h1>CLUEDO</h1>
+      <h2>Find out who's the murderer!</h2>
+
+      <section class="cards"> <!-- Section with all 3 cards-->
+        
+        <article class="killer-deck" id="killerCard"> <!-- Killers deck-->
+          <p class="deck-name">KILLER</p>
+
+          <div id="loader1"></div>
+
+          <img id="killerImage" src="" height="100" />
+          <p id="killerName"></p>
+          <p id="killerAge"></p>
+          <p id="killerOccupation"></p>
+          <p id="killerDescription"></p>
+          <p id="killerFavWeapon"></p>
+        </article>
+
+        <article class="weapon-deck" id="weaponCard"> <!-- Weapons deck-->
+          <p class="deck-name">WEAPON</p>
+
+          <div id="loader2"></div>
+
+          <img id="weaponImage" src="" height="100" />
+          <p id="weaponName"></p>
+          <p id="weaponWeight"></p>
+          <p id="weaponDamage"></p>
+        </article>
+
+        <article class="room-deck" id="roomCard"> <!-- Rooms deck-->
+          <p class="deck-name">ROOM</p>
+
+          <div id="loader3"></div>
+
+          <p id="roomName"></p>
+        </article>
+
+      </section>
+
+      <button onclick="revealMystery()" id="revealButton">REVEAL THE CRIME...</button>
```

Comment:
I'm curious why you chose this instead of
`document.getElementById("revealButton").addEventListener("click", revealMystery);`
Like you did with the cards